tests/Server/Workspace/SymbolTest.php

```diff
@@ -55,6 +55,7 @@ public function testEmptyQueryReturnsAllSymbols()
             new SymbolInformation('TestInterface',      SymbolKind::INTERFACE, $this->getDefinitionLocation('TestInterface'),                                ''),
             new SymbolInformation('test_function',      SymbolKind::FUNCTION,  $this->getDefinitionLocation('test_function()'),                              ''),
             new SymbolInformation('ChildClass',         SymbolKind::CLASS_,    $this->getDefinitionLocation('ChildClass'),                                   ''),
+            new SymbolInformation('TEST_PROPERTY',      SymbolKind::VARIABLE,  $this->getDefinitionLocation('TEST_PROPERTY'),                                ''),
```

Comment:
I'm curios about how it is returning null. A constructor cannot return anything?